xilem/src/view/button.rs

```diff
@@ -1,43 +1,45 @@
 // Copyright 2024 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
-use masonry::text::ArcStr;
+use masonry::text::StyleProperty;
 use masonry::widget;
 pub use masonry::PointerButton;
 
 use crate::core::{DynMessage, Mut, View, ViewMarker};
 use crate::{MessageResult, Pod, ViewCtx, ViewId};
+use crate::view::Label;
 
 /// A button which calls `callback` when the primary mouse button (normally left) is pressed.
 pub fn button<State, Action>(
-    label: impl Into<ArcStr>,
+    label: Label,
```

Comment:
I'm not a *massive* fan of all the callsites being `button(label("text"), ...)`. I.e. that most users will just want to use the unitary `Label` input.

I'm not sure what the right way to fix that would be. *Maybe* it would be to `impl Into<Label> for &str/Arc<str>` and use `impl Into<Label>`  here?

Reply:
I find a way to use `impl Into<Label>` after implementing `impl<T> From<T> for xilem::view::Label where T: Into<ArcStr>`